TwitchLib.Client/Manager/JoinedChannelManager.cs

```diff
@@ -21,9 +21,8 @@ public void AddJoinedChannel(JoinedChannel joinedChannel)
 
         public JoinedChannel GetJoinedChannel(string channel)
         {
-            var success = _joinedChannels.TryGetValue(channel, out var joinedChannel);
-
-            return success ? joinedChannel : null;
+            bool success = _joinedChannels.Any(x => x.Key.ToLower() == channel.ToLower());
```

Comment:
changing this makes no sense to me you could just pass in channel.ToLower()  to the TryGet which does the same but also in a threadsafe manner

Reply:
The change should be made to the dictionary - not to the get method. Simply passing in `StringComparer.OrdinalIgnoreCase` to the constructor of the dictionary will create a case insensitive set of keys